website/project/metadata/aea-prereg-2.json

```diff
@@ -0,0 +1,480 @@
+{
+    "name": "AEA_Preregistration",
+    "version": 2,
+    "description": 
+    "pages": [{
+        "id": "page1",
+        "title": "",
+        "contents": [{
+            "id": "general",
+            "type": "section",
+            "contents": [{
+                "required": true,
+                "type": "textfield",
+                "id": "title",
+                "label": "Trial Title",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_number",
+                "label": "AEA RCT Registration Number",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_date",
+                "label": "Registration Date",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "updated",
+                "label": "Last Updated",
+                "caption": "caption"
+            }],
+            "title": "General Information"
+        }, {
+            "type": "section",
+            "canRepeat": true,
+            "id": "location",
+            "contents": [{
+                "type": "select",
+                "required": true,
+                "options": [
```

Comment:
This is a very short and extremely... eclectic... list of countries. Can we get confirmation that no options have been left out? (or accidentally added in)